spec/es_spec_helper.rb

```diff
@@ -139,7 +139,7 @@ def put_alias(client, the_alias, index)
   end
 
   def clean_ilm(client)
-    client.get_ilm_policy.each_key {|key| client.delete_ilm_policy(name: key)}
+    client.get_ilm_policy.each_key { |key| client.delete_ilm_policy(name: key)  unless key =~ /history-ilm-policy/ }
```

Comment:
@robbavey thanks for sorting out this fix :)

One note, something to keep in mind in the futre, as this is the only place were we whilelist a policy: these clean* would be more resilient if they delete only the indices/aliases/ilm policies that are installed in the tests vs a "*".

For now it's not a big deal as they're only used in the ilm spec, but as ES has more and more content preloaded with it this list will grow.